internal/service/davinci/resource_flow.go

```diff
@@ -36,34 +48,34 @@ func ResourceFlow() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     true,
-				Description: "Deploy Flow after import. Flows must be deployed to be used.",
+				Description: "A boolean that specifies whether to deploy the flow after import. Flows must be deployed to become active.",
 			},
 			"name": {
 				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "Computed Flow Name after import. Matches 'name' in flow_json",
-			},
-			"environment_id": {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "PingOne Environment to import flow into.",
+				Description: "A string that identifies the flow name after import.",
 			},
 			"connection_link": {
 				Type:        schema.TypeSet,
 				Optional:    true,
 				Computed:    true,
-				Description: "Connections this flow depends on. flow_json connectionId will be updated to id matching name .",
+				Description: "Mappings to connections that this flow depends on.  Connections should be managed (with the `davinci_connection` resource) or retrieved (with the `davinci_connection` data source) to provide the mappings needed for this configuration block.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"id": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "Connection ID that will be used when flow is imported.",
+							Description: "A string that specifies the connector ID that will be applied when flow is imported.",
 						},
 						"name": {
 							Type:        schema.TypeString,
 							Required:    true,
-							Description: "Connection Name to match when updating flow_json connectionId.",
+							Description: "The connector name.  If `replace_import_connection_id` is also specified, this value is used when the flow is imported.  If `replace_import_connection_id` is not specified, the name must match that of the connector in the import file, so the connector ID in the `id` parameter can be updated.",
+						},
+						"replace_import_connection_id": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Connection ID of the connector in the import to replace with the connector described in `id` and `name` parameters.  This can be found in the source system in the \"Connectors\" menu, but is also at the following path in the JSON file: `[enabledGraphData|graphData].elements.nodes.data.connectionId`.",
```

Comment:
This would need a caveat that there isn't an intention of multiple connectors of the same type with different names used in a flow.